uint32 id
uint32 parent_drone_id
string name
string type
isaacs_server/topic_types[] topics
isaacs_server/topic_types[] services
